docs/update.md

```diff
@@ -0,0 +1,8 @@
+# Update
```

Comment:
Not sure if this deserves a separate markdown file. How about adding it to the installation docs?